e2e/cypress/integration/channel/message_spec.js

```diff
@@ -140,4 +141,66 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('MM-16730 Reply to an older message', () => {
+        cy.getCurrentChannelId().then((channelId) => {
+            // # Get yesterdays date in UTC
+            const yesterdaysDate = Cypress.moment().subtract(1, 'days').valueOf();
+
+            // # Post a day old message
+            cy.postMessageAs({sender: sysadmin, message: 'Hello from yesterday', channelId, createAt: yesterdaysDate}).
+                its('id').
+                should('exist').
+                as('yesterdaysPost');
+        });
+
+        // # Add two subsequent posts
+        cy.postMessage('One');
+        cy.postMessage('Two');
+
+        cy.get('@yesterdaysPost').then((postId) => {
+            // # Scroll up to yesterdays message
+            cy.get(`#post_${postId}`).scrollIntoView();
+
+            // # Open RHS comment menu
+            cy.clickPostCommentIcon(postId);
+
+            // # Upload an attachment to the reply
+            cy.fileUpload('#fileUploadInput').wait(TIMEOUTS.TINY);
+
+            // # Reply with the attachment
+            cy.postMessageReplyInRHS('A reply to an older post with attachment');
+
+            // # Get the latest reply post
+            cy.getLastPostId().then((replyId) => {
+                // * Verify that the reply is in the channel view with matching text
+                cy.get(`#post_${replyId}`).within(() => {
+                    cy.get('#postLink').should('be.visible').and('have.text', 'Commented on sysadmin\'s message: Hello from yesterday');
+                    cy.get(`#postMessageText_${replyId}`).should('be.visible').and('have.text', 'A reply to an older post with attachment');
+                    cy.get('.post-image__column').should('be.visible').and('have.text', 'mattermost-icon.png');
+                });
+
+                // * Verify that the reply is in the RHS with matching text
+                cy.get(`#rhsPost_${replyId}`).within(() => {
+                    cy.get('#postLink').should('not.be.visible');
+                    cy.get(`#postMessageText_${replyId}`).should('be.visible').and('have.text', 'A reply to an older post with attachment');
+                    cy.get('.post-image__column').should('be.visible').and('have.text', 'mattermost-icon.png');
```

Comment:
Currently the testcase fails here, because the element `.post-image__column` is not found. I guess you are referring to the element `.image-name`. Can you please recheck and update the locator correctly. 
![image](https://user-images.githubusercontent.com/1429138/62287133-df7a8500-b476-11e9-94a8-21ec189dca48.png)

Reply:
Same deal here with the prefernece to use an id for the `.post_image__column`. Depending on the react code, of course.

Reply:
Interesting, the file displays differently for me.

<img width="271" alt="Screen Shot 2019-08-01 at 9 57 25 am" src="https://user-images.githubusercontent.com/450704/62312392-e7b8cd00-b442-11e9-92ec-d5084d171584.png">

`.post-image__column` is visible however `.image-name` does not. I'll look into that and add a ID.

Reply:
Fixed in 3eba67f.